### DISCUSSION TITLE
Version of ffmpeg and go2rtc included?

### DISCUSSION THREAD

**Discussion by mjoy99** [blakeblackshear/frigate#6183] [Ask A Question]
I saw in the release notes for RC2 which version of go2rtc was included with frigte, but I couldn't see what version of ffmpeg is included.  what's the easiest way to see the versions for each of the installed dependencies?  I'm guessing I could just exec in to the container and do a version check but was wondering if there is documentation that covers this

**Comment by NickM-27** [comment]
Many parts of the documentation cover that go2rtc is 1.2.0

I don't think there's info on ffmpeg, but it is BtBn build 5.1.2
